groundstation/backend_api/models.py

```diff
@@ -13,14 +13,16 @@ class User(db.Model):
     slack_id = db.Column(db.String(128), nullable=True, unique=True)
     subscribed_to_slack = db.Column(db.Boolean, default=False)
     blacklisted_tokens = db.relationship('BlacklistedTokens', backref='user', lazy=True, cascade='all, delete-orphan')
+    creator = db.Column(db.String(128), nullable=True)
```

Comment:
Sorry if this might be too much work but I just realized that a better design pattern for the 'creator' column would be to reference the ID of the creator user through an SQL reference instead of a string. This makes it more efficient to see which admin user is responsible for each user and makes it easier to model the relationship.

See here: [https://docs.sqlalchemy.org/en/14/orm/self_referential.html](https://docs.sqlalchemy.org/en/14/orm/self_referential.html) for more information on how you can implement this. It is similar to how the 'Users' table is related to the 'Blacklisted' tokens table (see [Line 15](https://github.com/AlbertaSat/ex2_ground_station_website/blob/c221cd3f2a51bf11cf16ad7446bb7f900175a242/groundstation/backend_api/models.py#L15) and [Line 81](https://github.com/AlbertaSat/ex2_ground_station_website/blob/c221cd3f2a51bf11cf16ad7446bb7f900175a242/groundstation/backend_api/models.py#L81)) but instead of relating items between two different tables, you are instead relating items in the same table.

This will most likely also need a rewrite to the validator and the user.py API so that new users are matched appropriately with their creator. If this is too much we can save it later as a different task.